app/services/issue_output_document.rb

```diff
@@ -0,0 +1,23 @@
+class IssueOutputDocument
+  attr_accessor :appointment_summary
+
+  def initialize(appointment_summary)
+    @appointment_summary = appointment_summary
+  end
+
+  def call
+    return unless valid_email_address?
+
+    output_document = OutputDocument.new(@appointment_summary)
+    OutputDocumentMailer.issue(@appointment_summary, output_document.pdf)
+      .deliver_later
+  end
+
+  private
+
+  def valid_email_address?
+    @appointment_summary.validate
+    @appointment_summary.email_address.present? &&
+      @appointment_summary.errors[:email_address].blank?
```

Comment:
Good point. It's perhaps overkill checking the validation again here. Just checking if the email address is non-empty could be enough.